webui/react/src/hooks/useSettings.test.tsx

```diff
@@ -85,7 +85,7 @@ const config: hook.SettingsConfig<Settings> = {
       type: union([undefinedType, array(string)]),
     },
   },
-  storagePath: 'settings/normal',
+  storageKey: 'settings/normal',
```

Comment:
I would prefer if we had a consistent format for these (either all using `/` or all using `-`), but not a huge deal.